test/external-modules/multi-project/src/test/java/org/elasticsearch/multiproject/action/DeleteProjectActionTests.java

```diff
@@ -59,12 +59,12 @@ public void testDeleteNonExisting() throws Exception {
         var projects = randomList(1, 5, ESTestCase::randomUniqueProjectId);
         var deletedProjects = randomSubsetOf(projects);
         var state = buildState(projects);
-        var listener = ActionListener.assertAtLeastOnce(
-            ActionTestUtils.<AcknowledgedResponse>assertNoSuccessListener(e -> assertTrue(e instanceof IllegalArgumentException))
-        );
+        var listener = ActionListener.assertAtLeastOnce(ActionTestUtils.<AcknowledgedResponse>assertNoSuccessListener(e -> {
+            assertTrue(e instanceof IllegalArgumentException);
+        }));
```

Comment:
Nit: I don't think these lines need to be changed - changing the parameter to a lambda _block_ seems redundant.

Reply:
Yeah you are right. It was unintentional. Pushed 2716157da43dc28c76433f6753718980c6594a42